editor/editor_inspector.cpp

```diff
@@ -2844,7 +2848,7 @@ void EditorInspector::update_tree() {
 
 	Color sscolor = get_theme_color(SNAME("prop_subsection"), EditorStringName(Editor));
 
-	// Get the lists of editors to add the beginning.
+	// Add plugins that specifically request being added at the beginning
```

Comment:
```suggestion
	// Add plugins that specifically request being added at the beginning.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).